includes/wsuwp-shortcode-fundselector.js

```diff
@@ -443,6 +445,11 @@ function continueAction(event)
 			jQuery("#iDonateEmbed").attr("data-cash_default", des.amount);
 		}
 		else {
+			// Clear fields used by single designation
+			jQuery("#iDonateEmbed").attr("data-designation", null);
+			jQuery("#iDonateEmbed").attr("data-gift_arrays", null);
+			jQuery("#iDonateEmbed").attr("data-cash_default", null);
```

Comment:
Use the [`removeAttr()` function in jQuery](https://api.jquery.com/removeAttr/) instead, so it actually removes it. I think that will prevent any issues from iDonate trying to process a null value.